docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.    isa_qc.draw("mpl", idle_wires=0)
</pre><p>I feel it'd be more helpful to show the circuit <em>before</em> transpilation. So you can see the alternating RZ and CNOTs more clearly.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>